### PR TITLE
upload release to iTC/PM

### PR DESCRIPTION
**Pull request by rasom**
Jenkinsfile and other changes related to pipeline which will upload release apk/ipa. https://jenkins.status.im/job/status-react/job/upload_release/

status: ready 

<blockquote></blockquote>

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -34,4 +40,11 @@ platform :android do
     )
     slack(message: "New nightly build uploaded to Google Play")
   end
+  lane :nightly do
```
Comment (chadyj) [inline fastlane/Fastfile]: Should be `release`?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -34,4 +40,11 @@ platform :android do
...
+    upload_to_play_store(
+      track: "internal",
```
Comment (chadyj) [inline fastlane/Fastfile]: Release track is `alpha`

**Review by chadyj — CHANGES_REQUESTED** [review]
Very happy to see this progress :)

File: fastlane/Fastfile (hunk 1 above)
Comment (rasom) [reply fastlane/Fastfile]: fixed

File: fastlane/Fastfile (hunk 2 above)
Comment (rasom) [reply fastlane/Fastfile]: fixed

**Review by chadyj — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Pull request merged**